config/kafka.js

```
@@ -101,6 +104,11 @@ var _sendToKafkaConsumers = function (topic, host) {
 			if (message && message.topic === COURSES_TOPIC) {
 				userCoursesConsumer.messageReceived(message)
 			}
+
+			// call organizationExtension consumer
+			if (message && message.topic === ORG_EXTENSION_TOPIC) {
+				organizationExtensionConsumer.messageReceived(message)
```

ESLint

[error] 'organizationExtensionConsumer' is not defined.

(no-undef)